backend/siarnaq/api/teams/models.py

```diff
@@ -239,3 +239,28 @@ def get_avatar_url(self):
         )
         # Append UUID to public URL to prevent caching on avatar update
         return f"{public_url}?{self.avatar_uuid}"
+
+
+class ReferencePlayer(models.Model):
```

Comment:
pls rename to something that doesn't just sound like a team. 
ideas ---

related to the purpose of this all:
Requirement
ClassRequirement
PassRequirement
ReferencePlayerInfo
ReferencePlayerData

related to what this is specifically doing:
MatchChecker
MapWinChecker
etc ... keep riffing
Criteria (too generic?)